km/km_filesys.c

```diff
@@ -2975,7 +3043,9 @@ static int km_fs_recover_open_socket(char* ptr, size_t length)
          return -1;
       }
       if (bind(hostfd, sa, nt_sock->addrlen) < 0) {
-         km_warn("recover bind failed");
+         km_warn("recover bind failed %s:%d",
```

Comment:
This assumes address family is AF_INET.
Maybe an assert to let us know when we are being feed some other address family.